docs/Select-Canonical-Transcript.md

```diff
@@ -0,0 +1,92 @@
+# How Genome Nexus selects a canonical transcript
+
+## **Background**
+When a variant is sent to Genome Nexus, Genome Nexus calls **VEP** (Variant Effect Predictor)to get a range of annotations associated with the variant, including a list of potential transcripts.
+
+Genome Nexus then uses a set of predefined criteria to select the canonical transcript from this list of transcripts. The selection process is guided by the user-selected isoform override source and a prioritization logic.
+
+The canonical transcript is in the `transcriptConsequenceSummary` field, under the `annotation_summary` section of the response. To ensure the canonical transcript is returned, the `annotation_summary` field must be requested in the query sent to Genome Nexus.
+
+Example response structure:
+```
+"annotation_summary" {
+    "transcriptConsequenceSummary" {
+        ...
+    }
+}
+```
+
+### Available isoform override sources
+The isoform override source can be defined in `isoformOverrideSource` field, options are:
+- `mskcc`
+- `uniprot` (default)
+- `genome_nexus`
+- `ensembl`
+
+The list of canonical transcript isoform override for each gene can be found here, which is genereated by this [script](https://github.com/genome-nexus/genome-nexus-importer/blob/master/scripts/make_one_canonical_transcript_per_gene.py): 
+- GRCh37: https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/grch37_ensembl92/export/ensembl_biomart_canonical_transcripts_per_hgnc.txt
+- GRCh38: https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/grch38_ensembl95/export/ensembl_biomart_canonical_transcripts_per_hgnc.txt
+and 
+https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/grch38_ensembl92/export/ensembl_biomart_canonical_transcripts_per_hgnc.txt
+
+
+The original isoform override files can be found here:
+- `mskcc`:
+    - GRCh37: https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_oncokb_grch37.txt 
+    and 
+    https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_oncokb_grch37.txt
+    - GRCh38: https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_at_mskcc_grch38.txt
+    and
+    https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_oncokb_grch38.txt
+- `uniprot`:
+    - https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_uniprot.txt
+- `genome_nexus`:
+    - GRCh37: https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_genome_nexus_grch37.txt
+    - GRCh38: https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_genome_nexus_grch38.txt
+
```

Comment:
would be good to clarify for each of these where they originated from and why there were introduced